Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -967,17 +1000,17 @@ TilesetJsonLoader::loadTileContent(const TileLoadInput& loadInput) {
                         logTileLoadResult(pLogger, tileUrl, result.errors);
                         if (result.errors) {
                           return TileLoadResult::createFailedResult(
-                              std::move(pAssetAccessor),
-                              std::move(pCompletedRequest));
+                              pAssetAccessor,
+                              pCompletedRequest);
```

Comment:
These moves would be fine if the lambda were declared `mutable` and the `pCompletedRequest` were moved on capture.